modules/card-info/card-info.c

```diff
@@ -0,0 +1,105 @@
+#include "card-info.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+CardInfo initCardInfo() {
+    CardInfo cardInfo;
+    cardInfo.identificationTraitsSize = 0;
+    cardInfo.circumstancesSize = 0;
+
+    return cardInfo;
+}
+
+void getCardInfoFromUser(CardInfo *cardInfo) {
+    char userAnswer = ' ';
+    int year;
+    int month;
+    int day;
+
+    printf("Enter person's name: ");
+    fgets(cardInfo->name, PERSON_NAME_LEN, stdin);
+    cardInfo->name[strcspn(cardInfo->name, "\n")] = 0;
```

Comment:
You can use such macro:
```c
#define FORMAT(S) "%" #S "s"
#define VARIABLE_LEN_STR(S) FORMAT(S)
// ...
int retValue = scanf(VARIABLE_LEN_STR(PERSON_NAME_LEN), cardInfo->name);

if (retValue == 1) {
    // scanf successful
} else {
    // handle scanf error
}
```

This would be more optimal. Make the same in other places.